src/main/java/com/mraof/minestuck/util/StartingModusManager.java

```diff
@@ -0,0 +1,96 @@
+package com.mraof.minestuck.util;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+import com.google.gson.stream.JsonReader;
+import com.mraof.minestuck.Minestuck;
+import com.mraof.minestuck.inventory.captchalogue.ModusType;
+import com.mraof.minestuck.inventory.captchalogue.ModusTypes;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.server.packs.resources.Resource;
+import net.minecraft.server.packs.resources.ResourceManager;
+import net.minecraft.server.packs.resources.SimplePreparableReloadListener;
+import net.minecraft.util.profiling.ProfilerFiller;
+import net.minecraftforge.event.AddReloadListenerEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Mod.EventBusSubscriber(bus = Mod.EventBusSubscriber.Bus.FORGE)
+public class StartingModusManager extends SimplePreparableReloadListener<List<String>>
+{
+	private static final Logger LOGGER = LogManager.getLogger();
+	private static final Gson GSON = new GsonBuilder().create();
+	
+	public static final String PATH = "minestuck/config/starting_modus.json";
+	
+	private static StartingModusManager INSTANCE;
+	private List<ModusType<?>> startingModusTypes;
+	
+	@Override
+	protected List<String> prepare(ResourceManager resourceManager, ProfilerFiller profiler)
+	{
+		ResourceLocation location = new ResourceLocation(Minestuck.MOD_ID, PATH);
+		if(!resourceManager.hasResource(location))
+			return null;
+		
+		try(Resource resource = resourceManager.getResource(location);
```

Comment:
It may be nice to load lists across multiple namespaces and then merge them together into one. Doing so would better support potential addons adding new modus types by letting them add their types through their own data file without having to override minestucks data file. `RecipeGeneratedCostHandler` does exactly that for its data. Consider this a softer request that I'll likely do myself later if it doesn't get done in this PR.